<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>downloadWindow</class>
 <widget class="QMainWindow" name="downloadWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dooble Web Browser: Downloads</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="_2">
    <property name="spacing">
     <number>5</number>
    </property>
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page1">
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>5</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QCheckBox" name="bitRateCheckBox">
            <property name="text">
             <string>Show Bit Rates</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="table">
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="horizontalScrollMode">
           <enum>QAbstractItemView::ScrollPerPixel</enum>
          </property>
          <property name="showGrid">
           <bool>false</bool>
          </property>
          <property name="columnCount">
           <number>1</number>
          </property>
          <attribute name="horizontalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <column/>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page2">
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <property name="spacing">
         <number>5</number>
        </property>
        <property name="leftMargin">
         <number>5</number>
        </property>
        <property name="topMargin">
         <number>5</number>
        </property>
        <property name="rightMargin">
         <number>5</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>D&amp;ownload URL</string>
          </property>
          <property name="buddy">
           <cstring>urlLineEdit</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="urlLineEdit"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>5</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="clearPushButton">
            <property name="text">
             <string>Clea&amp;r</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="cancelPushButton">
            <property name="text">
             <string>C&amp;ancel</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="downloadPushButton">
            <property name="text">
             <string>&amp;Download</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="dgenericsearchwidget" name="searchLineEdit"/>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="closePushButton">
        <property name="text">
         <string>&amp;Close</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="clearListPushButton">
        <property name="text">
         <string>Clear &amp;List</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="enterUrlPushButton">
        <property name="text">
         <string>Enter &amp;URL</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>dgenericsearchwidget</class>
   <extends>QLineEdit</extends>
   <header>dgenericsearchwidget.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>bitRateCheckBox</tabstop>
  <tabstop>table</tabstop>
  <tabstop>urlLineEdit</tabstop>
  <tabstop>clearPushButton</tabstop>
  <tabstop>cancelPushButton</tabstop>
  <tabstop>downloadPushButton</tabstop>
  <tabstop>searchLineEdit</tabstop>
  <tabstop>closePushButton</tabstop>
  <tabstop>clearListPushButton</tabstop>
  <tabstop>enterUrlPushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
